volatility3/framework/plugins/windows/pe_symbols.py

```diff
@@ -0,0 +1,437 @@
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+
+import io
+import logging
+from typing import Dict, Tuple, Optional, List
+
+import pefile
+
+from volatility3.framework import interfaces, exceptions
+from volatility3.framework import renderers, constants
+from volatility3.framework.configuration import requirements
+from volatility3.framework.renderers import format_hints
+from volatility3.framework.symbols import intermed
+from volatility3.framework.symbols.windows import pdbutil
+from volatility3.framework.symbols.windows.extensions import pe
+from volatility3.plugins.windows import pslist, vadinfo, modules
+
+vollog = logging.getLogger(__name__)
+
+# used for special handling of the kernel PDB file. See later notes
+os_module_name = "ntoskrnl.exe"
```

Comment:
Uhhh, could this not live in `constants.windows` or just... somewhere rather than floating in the middle of a plugin file?

Reply:
I will move it inside the class. Its just the convention I used since the kernel file can have multiple names even in the same build (see the comments about the PDB searching).

Reply:
Ok, I don't believe it changes in this plugin, so... feels like a constant.  Happy for it to live somewhere you're happy with, but the module isn't the right place.

Reply:
moved into the class.